R/autoplot.pop_data.R

```diff
@@ -6,68 +6,109 @@
 #' @param log whether to show antibody responses on logarithmic scale
 #' @param strata the name of a variable in `pop_data` to stratify by (or `NULL` for no stratification)
 #' @param ... unused
+#' @param type an option to choose type of chart, defaults to density chart
 #'
 #' @return a [ggplot2::ggplot] object
 #' @export
 #' @examples
 #' library(dplyr)
 #' library(ggplot2)
 #'
+#'
 #' xs_data <- "https://osf.io/download//n6cp3/" %>%
 #' load_pop_data() %>%
 #' clean_pop_data()
 #'
 #' xs_data %>% autoplot(strata = "Country")
 #'
+
 autoplot.pop_data = function(
     object,
     log =  FALSE,
+    type = 'density',
     strata = NULL,
     ...)
 {
+  if(type == 'age-scatter')
```

Comment:
- [ ] please throw an error if "type" isn't one of the options we have implemented

Reply:
This has been implemented